packages/integrations/mdx/test/mdx-page.test.js

```diff
@@ -0,0 +1,28 @@
+import mdx from '@astrojs/mdx';
+
+import { expect } from 'chai';
+import { parseHTML } from 'linkedom'
+import { loadFixture } from '../../../astro/test/test-utils.js';
+
+describe('MDX Page', () => {
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({
+			root: new URL('./fixtures/mdx-page', import.meta.url),
+			integrations: [
+				mdx()
+			]
+		});
+		await fixture.build();
+	});
+
+	it('builds', async () => {
```

Comment:
Same as above, would be nice to have a quick `dev` test to catch regressions

Reply:
Added in [fac204a](https://github.com/withastro/astro/pull/3706/commits/fac204aaadd1e12751d0d89e0d678de907939742)